src/react-count-to.js

```diff
@@ -32,11 +33,10 @@ const CountTo = React.createClass({
   },
 
   componentWillReceiveProps(nextProps) {
-    const props = this.props;
+    const { from, to } = this.props;
 
-    // no need to restart animation if onComplete / digits / className changes
-    if (nextProps.to !== props.to || nextProps.from !== props.from
-    || nextProps.speed !== props.speed || nextProps.delay !== props.delay) {
+    // restart only if to / from changes
```

Comment:
I would say that this comment is unnecessary since the code is very clear.
I try to avoid comments as much as I can :)